### ISSUE TITLE
ESP32 example?

### ISSUE THREAD

**Issue by DatanoiseTV** [essej/aoo#8]
I just stumbled upon this and was wondering when you are planing to release the ESP32
example.

I can test it with wireless and wired Ethernet and have access to multiple S3 and P4 boards.

**Comment by Spacechild1** [comment]
Hi, I think you're in the wrong Repo :) This is an (old) fork by @essej, the official GitHub mirror is https://github.com/Spacechild1/aoo. Unfortunately, Google ranks Jesse's fork much higher than my own mirror...

Also, consider using the official issue tracker: https://git.iem.at/aoo/aoo/-/issues.

As for the ESP32 example, here's one project I did with the ESP32: https://git.iem.at/aoo/aoo/-/issues. I want to simplify it and add it as an example to the AOO repo. So far, it only demonstrates how to receive audio streams on the ESP32, but I also want to add an example for using the ESP32 as a microphone. Note that I'm using the [Olimex ESP32-ADF board](https://www.olimex.com/Products/IoT/ESP32/ESP32-ADF/open-source-hardware) which has 8 MB of PSRAM (of which, I think, up to 4 MB are actually usable).

If you have more questions, let's continue at the AOO repo :)

**Comment by essej** [comment]
Yeah, sorry about that... hopefully soon I won't be using such a diverged fork! I should make a comment in the README at the top that makes this more obvious...

**Comment by Spacechild1** [comment]
> Yeah, sorry about that

Not your fault!

> I should make a comment in the README at the top that makes this more obvious...

That would be cool!

**Comment by Spacechild1** [comment]
ping @essej. I just had another case where someone accidentally used this repo instead of the official one.

(I can't blame the people, because Google decides to rank your repo way above my official mirror :-D)

**Comment by Spacechild1** [comment]
Thanks :)
